docs/topics/modules/creating_new_application.adoc

```diff
@@ -28,7 +28,7 @@ ifeval::["{context}" == "hello-world"]
 +
 image::choose_mission_runtime.png[Choose mission and runtime]
 +
-. In the *Select Pipeline* section, select the first option, then click the blue arrow to continue to the next step.
+. In the *Select Pipeline* section, select the first option, then click the blue arrow to continue to the next step. The {osio} pipelines create repeatable, incrementally improving processes that test and deploy the code from creation to execution.
```

Comment:
Do we need this addition in this step? It doesn't seem particularly useful to the step, just sort of marketing-speak-ish?

Reply:
Removing this for now, agree on the marketing-speak-ish observation, but more so for consistency. OTOH, in the creating application module, we are not explaining the why anywhere. Should this be rethought? Should we not have at least a line to explain the whys in the steps? We ask the user to do 1,2,3...without telling him why he needs to do them, and this one of the primary areas of focus in OSIO.